pages/profile/[username].tsx

```diff
@@ -14,50 +15,79 @@ import Head from "next/head"
 
 import Lottie from "react-lottie"
 import animationData from "../../lottie/sad.json"
-const defaultOptions = {
+import {ProfileSerializer} from "types/backend"
+const defaultOptions: {
+    loop: boolean
+    autoplay: boolean
+    animationData: any | null
+} = {
     loop: true,
     autoplay: true,
     animationData: animationData,
     // rendererSettings: {
     // 	preserveAspectRatio: 'xMidYMid slice'
     // }
 }
+export interface Auth {
+    token?: string | null
+    profile?: any | null
+    loading?: boolean
+}
+export interface RUser {
+    loading?: boolean
+    user?: any | null
```

Comment:
user is initialize with value  "NOT FOUND" (string). Line 92.

Reply:
I tried, but it show error, string override non-primitive values ig, till then i initialize user value null instead of "NOT FOUND" and use ProfileSerializer | null.

Reply:
Sorry, my bad...you are correct we need to use `ProfileSerializer | null` here. The backend returns the string only the error code is 404, so the user is set as null.